src/main/java/Planes/Plane.java

```diff
@@ -19,19 +21,18 @@ public String getModel() {
         return model;
     }
 
-    public int getMS() {
+    public int getMaxSpeed() {
         return maxSpeed;
     }
 
-    public int Get_Max_Flight_Distance() {
+    public int GetMaxFlightDistance() {
```

Comment:
import lombok.Getter;
import lombok.Setter;
https://www.baeldung.com/intro-to-project-lombok

Reply:
done